engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/ExecutionStore.scala

```diff
@@ -22,19 +21,22 @@ object ExecutionStore {
       case _ => None
     }
 
-    new ExecutionStore(keys.flatten.map(_ -> NotStarted).toMap)
+    new ExecutionStore(keys.flatten.map(_ -> NotStarted).toMap, keys.nonEmpty)
   }
 }
 
-case class ExecutionStore(private val store: Map[JobKey, ExecutionStatus]) {
+case class ExecutionStore(private val statusStore: Map[JobKey, ExecutionStatus], hasNewRunnables: Boolean) {
```

Comment:
No it's not but even without modifying it, it shouldn't be randomly read without it being thought through, because of how large it can be and the performance cost associated.